build/taskctl/contexts.yaml

```diff
@@ -13,16 +12,18 @@ contexts:
         - -w
         - /app
         - --env-file
-        - envfile        
-        - amidostacks/runner-pwsh-golang:0.4.60-stable
+        - envfile
+        - ensonostackseuweirdfmu.azurecr.io/ensono/eir-golang:1.1.91
```

Comment:
Could you try the latest main images e.g.: `ensono/eir-golang:224` so we know we're in good shape

Reply:
Also take a look at the contexts in this file, can we remove the -e for Module Path, and -NoProfile stuff?

Reply:
And bump all images to the latest 224 🙂